Core/Utilities/Pagination/PaginationParameters.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace WebNetSample.Core.Pagination;
+
+public abstract class PaginationParameters
+{
+    const int maxPageSize = 48;
```

Comment:
It is a good practice in C# to structure your class level items in the following order:
1. constants (does not matter what accessor they have)
2. private fields
3. constructor
4. (public, private) properties
5. methods

Let's make the same order here, please :)